src/main/kotlin/com/sourcegraph/config/ConfigUtil.kt

```diff
@@ -141,11 +139,11 @@ object ConfigUtil {
 
     return try {
       val text = customConfigContent ?: getSettingsFile(project).readText()
-      val config = ConfigFactory.parseString(text).resolve()
+      val config = JsonParser.parseString(text).asJsonObject
```

Comment:
ConfigFactory does not render json. The rendered config contained the comment with # instead of //. Using JsonParser fixes the problem.

Reply:
Other option would be adding `.setComments(false)` in the current code.
I thought Lightened Config was a bit more forgiving when parsing e.g. additional commas.
Maybe we can write simple test for that and check how those corner cases looks in case of both approaches?

Reply:
fixed